roles/neutron-common/templates/etc/neutron/policy.json

```diff
@@ -36,8 +46,11 @@
     "get_network:provider:physical_network": "rule:admin_only",
     "get_network:provider:segmentation_id": "rule:admin_only",
     "get_network:queue_id": "rule:admin_only",
+    "get_network_ip_availabilities": "rule:admin_only",
+    "get_network_ip_availability": "rule:admin_only",
     "create_network:shared": "rule:admin_only",
     "create_network:router:external": "rule:admin_only",
+    "create_network:is_default": "rule:admin_only",
```

Comment:
I looked into this feature in mitaka and think we should move this to next release if there is no customer urgent customer requirement.